addons/web/static/tests/fields/basic_fields_mobile_tests.js

```diff
@@ -173,6 +173,54 @@ QUnit.module('basic_fields', {
 
         form.destroy();
     });
+
+    QUnit.module('FieldDateRange');
+
+    QUnit.test('date field: toggle daterangepicker then scroll', async function (assert) {
+        assert.expect(5);
+        const scrollEvent = new UIEvent('scroll');
+
+        function scrollAtHeight(height) {
+            window.scrollTo(0, height);
+            document.dispatchEvent(scrollEvent);
+        }
+        this.data.partner.fields.date_end = {string: 'Date End', type: 'date'};
+
+        var form = await createView({
+            View: FormView,
+            model: 'partner',
+            data: this.data,
+            arch: '<form>' +
+                    '<field name="date" widget="daterange" options="{\'related_end_date\': \'date_end\'}"/>' +
+                    '<field name="date_end" widget="daterange" options="{\'related_start_date\': \'date\'}"/>' +
+                '</form>',
+            session: {
+                getTZOffset: function () {
+                    return 330;
+                },
+            },
+        });
+
+        // Check date range picker initialization
+        assert.containsN(document.body, '.daterangepicker', 2,
+            "should initialize 2 date range picker");
+        assert.strictEqual($('.daterangepicker:first').css('display'), 'block',
```

Comment:
After having thought long and hard, could you please remove these two asserts?
I am not fully convinced that we want it opened or not.
Even if, this assert is already present in https://github.com/odoo/odoo/commit/cda93a5c19e564b00da4a3c97aceabbcb7059ed7#diff-12238b9915b0963dd270beb854d85d0a094376093b1b06781a2010cb7d48e4c4R2730

Reply:
@nboulif

Reply:
Done :+1: